example/Oauth2/getmaterials.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: johnny
+ * Date: 8/14/18
+ * Time: 3:46 PM
+ */
+require './common.php';
+
+// generate an access token
+try {
+  $a = $client->generateAccessTokenClientCredentialGrant();
+  $client->accessToken = $a->access_token;
+
+} catch (\Exception $e) {
+  // printing error on screen
+  echo 'Exception: '. $e->getMessage();
+  exit();
+}
+
+try {
+  $materials = $client->getMaterials();
+} catch (\Exception $e) {
+  // printing error on screen
+  echo 'Exception: '. $e->getMessage();
+  exit();
+}
+
```

Comment:
missing `header("Content-type: application/json");` since I assume we'e returning json?

Reply:
unless this is just test sample thingy

Reply:
yea this is a quick example for testing